resources/views/errors/404.blade.php

```diff
@@ -0,0 +1,70 @@
+<html>
+    <head>
+        <title>RoboHome | Login</title>
+        <script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
+        <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+        <!-- Latest compiled and minified JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <style>
+          /* Error Page Inline Styles */
+          body {
+            padding-top: 20px;
+          }
+          /* Layout */
+          .jumbotron {
+            font-size: 21px;
+            font-weight: 200;
+            line-height: 2.1428571435;
+            color: inherit;
+            padding: 10px 0px;
+          }
+          /* Everything but the jumbotron gets side spacing for mobile-first views */
+          .masthead, .body-content, {
+            padding-left: 15px;
+            padding-right: 15px;
+          }
+          /* Main marketing message and sign up button */
+          .jumbotron {
+            text-align: center;
+            background-color: transparent;
+          }
+          .jumbotron .btn {
+            font-size: 21px;
+            padding: 14px 24px;
+          }
+          /* Colors */
+          .green {color:#5cb85c;}
+          .orange {color:#f0ad4e;}
+          .red {color:#d9534f;}
+        </style>
+        <script type="text/javascript">
+          function loadDomain() {
+            var display = document.getElementById("display-domain");
+            display.innerHTML = document.domain;
+          }
+        </script>
+    </head>
+    <body>
+       <div class="container">
+          <div class="jumbotron">
+            <h1><i class="glyphicon glyphicon-exclamation-sign"></i></h1>
+            <h2>404 Not Found</h2>
+            <p class="lead">We couldn't find what you're looking for on <em><span id="display-domain"></span></em>.</p>
```

Comment:
I'm not sure the `<em><span id="display-domain"></span></em>` part works.  When I get to the 404 page all I see is the following:

<img width="411" alt="screen shot 2017-10-04 at 18 54 46" src="https://user-images.githubusercontent.com/15161181/31203530-8a6f8d7e-a935-11e7-9787-34fa29ebff02.png">

There are several really good Laravel Dusk test that should be added.  The test should:

1. Make sure the back button exists
2. Make sure the go home button exists
3. Make sure the title of the page is correct
4. Make sure the page gets loaded for a bad URL

Given that access to this page isn't behind a login wall, it shouldn't be too hard to test, see examples in this project here: https://github.com/dbudwin/RoboHome-Web/blob/master/tests/Browser/IndexTest.php